src/base/bittorrent/session.h

```diff
@@ -63,7 +63,16 @@ enum MaxRatioAction
 {
     Pause,
     Remove,
-    EnableSuperSeeding
+    EnableSuperSeeding,
+    DeleteFiles,
+    DeleteFolder
```

Comment:
I would assign enum values explicitly to be sure they will not change accidentally. Then you can also reorder them logically (to keep all delete options together).